packages/common/src/projects/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * As of typescript 3.8, we can use `export * as namespace`
+ * to package up the contents of a module under a specific name
+ */
+export * as HubProjects from "./HubProjects";
```

Comment:
This is how we get `HubProjects` as a module importable as `import {HubProjects} from '@esri/hub-common'`